rush-plugins/rush-http-build-cache-plugin/src/HttpBuildCacheProvider.ts

```diff
@@ -39,15 +39,14 @@ export interface IHttpBuildCacheProviderOptions {
   cacheKeyPrefix?: string;
   isCacheWriteAllowed: boolean;
   pluginName: string;
-  rushProjectRoot: string;
+  rushRepoRoot: string;
```

Comment:
`rushWorkspaceRoot` or `rushJsonFolder`.

We expressly support `rush.json` not being at the root of the Git repository.